cmd/aks-periscope/aks-periscope.go

```diff
@@ -14,83 +15,91 @@ import (
 
 func main() {
 	zipAndExportMode := true
-	exporter := &exporter.AzureBlobExporter{}
+	exporters := []interfaces.Exporter{}
+	clusterType := os.Getenv("CLUSTER_TYPE")
+
+	exporters = append(exporters, &exporter.AzureBlobExporter{})
+	if strings.EqualFold(clusterType, "connectedCluster") {
+		exporters = append(exporters, &exporter.LocalMachineExporter{})
+	}
+
 	var waitgroup sync.WaitGroup
 
 	err := utils.CreateCRD()
 	if err != nil {
 		log.Printf("Failed to create CRD: %+v", err)
 	}
+	for _, exporter := range exporters {
```

Comment:
What is this change?

Reply:
Ok will fix this then.

Reply:
Actually I think it could be fine as the user could specify multiples exporters.
However, I don't understand why for each exporter we run the collectors and the diagnosers? To me, it should be run collectors --> run diagnosers --> run exporters

Reply:
I think this is ok if they user wants to use multiple exporters.
However, collectors and diagnosers should run once once for all exporter
the loop should only do something like
```go
for _, exporter := range exporters {
  exporter.Export()
}
```